lib/service-data/service-data.js

```diff
@@ -512,8 +512,74 @@ external.setInstanceProperty = (id, property, value) => {
     })
```

Comment:
Knowledge sharing reminder: what is happening here!

Reply:
Yep, we'll continue to deep-dive